src/lib/autocomplete/autocomplete.spec.ts

```diff
@@ -611,6 +611,24 @@ describe('MdAutocomplete', () => {
       });
     }));
 
+    it('should not call the default input behaviour when an option is selected with ENTER',
+        async(() => {
+      fixture.whenStable().then(() => {
+        fixture.componentInstance.trigger._handleKeydown(DOWN_ARROW_EVENT);
+
+        fixture.whenStable().then(() => {
+          fixture.detectChanges();
```

Comment:
Done. I've removed the extra `detectChanges` but had to keep the `fixture.whenStable()` so that the test could simulate the "open" and "select first option" action before it press the enter key